grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -709,6 +709,9 @@ class OrderService {
             log.info "Order line items " + orderItems
 
             Order order = Order.get(orderId)
+            if (!isOrderEditable(order, user)) {
+                throw new UnsupportedOperationException("You do not have permissions to perform this action")
```

Comment:
How about translating it? We throw this exception in many places using the translated version 
```suggestion
                throw new UnsupportedOperationException("${g.message(code: 'errors.noPermissions.label')}")
```